.travis.yml

```diff
@@ -2,7 +2,9 @@ language: ruby
 cache: bundler
 sudo: required
 dist: trusty
-before_install: gem install bundler
+before_install:
+  - gem install bundler
+  - mysql -u root -e 'create database globalize_test character set utf8;' >/dev/null
```

Comment:
What do think about to put the  `character set utf8` argument directly here? 
https://github.com/globalize/globalize/blob/master/test/support/database.rb#L58